java-cacerts.yaml

```diff
@@ -14,7 +14,7 @@ package:
 environment:
   contents:
     packages:
-      - ca-certificates
+      - ca-certificates=${{package.full-version}}
```

Comment:
Why did you use full-version here rather than just version?
full version includes epoch.  So you can't change this package at _all_ without bumping ca-certificates.
Maybe thats what you meant by ' lockstep against ca-certs', but it feels overly lockstep.

Reply:
@smoser we had to revert certificate removal in an empoch. thus yes this is intentional to propagate. See for example https://github.com/wolfi-dev/os/commit/a674f13e78d98d7e33e38f82acb04450c25bb43e : 
- https://github.com/wolfi-dev/os/pull/39066

Which broke production deployments.